app/controllers/api/articles_controller.rb

```diff
@@ -3,9 +3,9 @@ class Api::ArticlesController < ApplicationController
   before_action :role_authenticator, only: %i[create]
 
   def index
-    articles = Article.all.most_recent
+    articles = Article.all.sort_by { |article| article[:updated_at] }
     if current_user&.journalist?
-      articles_by_journalist = Article.where(user_id: current_user.id).most_recent
+      articles_by_journalist = Article.where(user_id: current_user.id).sort_by { |article| article[:updated_at] }.reverse
```

Comment:
If you guys check how we made the `.most_recent`, you could do the same with this :D

Reply:
Also Fixed.